src/common.c

```diff
@@ -2182,7 +2190,7 @@ void ofi_get_list_of_addr(const struct fi_provider *prov, const char *env_name,
 	}
```

Comment:
`addr_entry->mtu` is unitialized in this case

Reply:
I'm fairly unfamiliar with Windows, which is where this code is used. Given what I see in `include/windows/config.h`, it's not clear to me that this code will actually be compiled, but I'm probably missing something. In any case, initializing `addr_entry->mtu` to -1 suffices to make the program correct, although the standard default MTU will then be assumed.